lineage/BaumWelch.py

```diff
@@ -97,11 +101,14 @@ def do_M_T_step(tHMMobj, MSD, betas, gammas):
         gamma_array = gammas[num]
 
         # local T estimate
-        numer_estimate += get_all_zetas(lineageObj, betas[num], MSD[num], gamma_array, tHMMobj.estimate.T)
-        denom_estimate += sum_nonleaf_gammas(lineageObj, gamma_array)
+        numer_estimate = get_all_zetas(lineageObj, betas[num], MSD[num], gamma_array, tHMMobj.estimate.T)
```

Comment:
Are you sure this is right? You're only going to end up estimating based on the last iteration of the loop...

Reply:
Sorry, I was messing with it and forgot to return it back to the original version and accidentally committed.